app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Booking Details</h1>
+
+<div class="cards">
+  <p><%= "Superpower ID: "%><%= link_to "#{@booking.superpower_id}", "/superpower_path" %></p>
```

Comment:
you don't need a <%= %> tag for the superpower ID part, as it's not ruby code

Reply:
done :)